### PR TITLE
More fixes of Upload screen

### PR DESCRIPTION
**Pull request by kiroj305**
- title center align
- show chosen file name and size in proper position
- fix upload error popup ui
- optimize upload screen composition

### CODE REVIEW

File: lib/screens/upload_screen.dart [hunk 1]
```diff
@@ -193,78 +179,78 @@ class _ErrorMessageWidget extends StatelessWidget {
...
+                Expanded(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Text("• ${kFileSizeLimitInGB}GB Limit",
+                          style: Theme.of(context)
+                              .textTheme
+                              .bodyText2!
+                              .copyWith(color: Colors.white, fontSize: 16.sp, fontWeight: FontWeight.w800)),
+                      Text("• Image, Video, 3D or Audio",
```
Comment (kjawadDeveloper2) [inline lib/screens/upload_screen.dart]: Make constants.

File: lib/screens/upload_screen.dart [hunk 2]
```diff
@@ -193,78 +179,78 @@ class _ErrorMessageWidget extends StatelessWidget {
...
+                          style: Theme.of(context)
+                              .textTheme
+                              .bodyText2!
+                              .copyWith(color: Colors.white, fontSize: 16.sp, fontWeight: FontWeight.w800)),
+                      Text("• One file per upload",
```
Comment (kjawadDeveloper2) [inline lib/screens/upload_screen.dart]: Make constants.

**Review by mustafapylons — APPROVED** [review]
Looks good to me.

File: lib/screens/upload_screen.dart (hunk 2 above)
Comment (kiroj305) [reply lib/screens/upload_screen.dart]: Done @kjawadDeveloper2

**Pull request merged**